commands/cli.go

```diff
@@ -140,6 +145,10 @@ func CreateCli(version string, mixPanelClientId string) *cli.App {
 					Usage: "Only inspect resources older than this specified value. Can be any valid Go duration, such as 10m or 8h.",
 					Value: "0s",
 				},
+				&cli.BoolFlag{
+					Name:  "list-unaliased-kms-keys",
+					Usage: "Delete KMS keys that do not have aliases associated with them.",
```

Comment:
I was waffling back and forth and then settled on "give them the option". I chose that option because I wasn't sure if people were using `list` as a kind of `terraform plan` esque step. The thought being - if the default for the nuke is to only delete KMS keys with aliases, then the default for `list` should only display those resources.

If that's not the use/intended use of `list`, I'm fine always showing all keys (with and w/o aliases) by default. WDYT?

Reply:
I think that makes sense. I typically don't bother with a plan and just don't approve the nuke after it has shown the output, but I can see the use case.  Let's leave the flag in